### PR TITLE
[Copilot] Update readme, rename samples, add two samples

### PR DESCRIPTION
**Pull request by encimita**

### CODE REVIEW

File: samples/AzureOpenAI/1-GetStarted/DraftProject.Page.al [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            field(ProjectDescriptionField; InputProjectDescription)
+            {
+                ShowCaption = false;
+                MultiLine = true;
+                InstructionalText = 'Please describe the project or work to be done that needs to be converted into tasks';
+            }
+        }
+
+        area(PromptOptions)
+        {
```
Comment (juliamakulec) [inline samples/AzureOpenAI/1-GetStarted/DraftProject.Page.al:25]: Minor - why do we need an empty area?

File: samples/AzureOpenAI/3-SuggestJob/Advanced_suggestjob_sourcecode_1pauth.zip
Comment (juliamakulec) [inline samples/AzureOpenAI/3-SuggestJob/Advanced_suggestjob_sourcecode_1pauth.zip:1]: Are the .zip files updated?

File: samples/AzureOpenAI/ReadMe.md [hunk 2]
```diff
@@ -1,19 +1,57 @@
...
+
+This extension is ready for you to explore. If you are NOT using Managed AI Resources, you will get an error you get an error similar to "The authorization is not configured". You need to change the line in `DraftProject.Page.al`, and instead of `AzureOpenAI.SetManagedResourceAuthorization` call `AzureOpenAI.SetAuthorization` specifying your own Azure OpenAI endpoint, deployment, and API key. The endpoint typically has the format of: `https://<resource name>.openai.azure.com/`.
+
+After you install the extension, you will be able to access its logic from the existing Base Application page `page 89 "Job List"`.
+
+
+### 2-ItemSubstitution
+This folder contains one of the early samples that uses the AL Copilot Toolkit. It demonstrates how to use the PromptDialog page type and its special capabilities, as well as how to set up and tweak the parameters to call Azure Open AI from AL.
+
+This extension is ready for you to explore. Before you install it, you need to modify the file `SecretsAndCapabilitiesSetup.Codeunit.al`, and make sure Isolated Storage is set up with your Azure OpenAI endpoint, deployment, and API key.
```
Comment (juliamakulec) [inline samples/AzureOpenAI/ReadMe.md:30]: Aren't we switching everywhere for Managed resources?

File: samples/AzureOpenAI/ReadMe.md [hunk 3]
```diff
@@ -1,19 +1,57 @@
...
+
+If you are NOT using Managed AI Resources, you will get an error you get an error similar to "The authorization is not configured". You need to change the line in `GenerateItemSubProposal.Codeunit.al`, and instead of `AzureOpenAI.SetManagedResourceAuthorization` call `AzureOpenAI.SetAuthorization`.
+
+After you install the extension, you will be able to access its logic from the existing Base Application page `page 30 "Item Card"`.
+
+
+### 3-SuggestJob
+This folder contains one of the early samples that uses the AL Copilot Toolkit. It demonstrates how to use the PromptDialog page type and its special capabilities, as well as how to set up and tweak the parameters to call Azure Open AI from AL.
+
+This extension is ready for you to explore. Before you install it, you need to modify the file `SecretsAndCapabilitiesSetup.Codeunit.al`, and make sure Isolated Storage is set up with your Azure OpenAI endpoint, deployment, and API key.
```
Comment (juliamakulec) [inline samples/AzureOpenAI/ReadMe.md:40]: Same comment as above

File: samples/AzureOpenAI/ReadMe.md (hunk 2 above)
Comment (encimita) [reply samples/AzureOpenAI/ReadMe.md:30]: Yes, but that still requires endpoint, deployment, and API key. At least for the moment being.

File: samples/AzureOpenAI/ReadMe.md (hunk 3 above)
Comment (encimita) [reply samples/AzureOpenAI/ReadMe.md:40]: Same reply as above!

File: samples/AzureOpenAI/3-SuggestJob/Advanced_suggestjob_sourcecode_1pauth.zip
Comment (encimita) [reply samples/AzureOpenAI/3-SuggestJob/Advanced_suggestjob_sourcecode_1pauth.zip:1]: Good catch! Updated them now.

**Review by juliamakulec — APPROVED** [review]

**Pull request merged**